src/CHIPGraph.hh

```diff
@@ -645,20 +700,34 @@ public:
   }
 };
 
-class CHIPGraphExec : public hipGraphExec {
+class GraphNative {
```

Comment:
This could have a class comment which states what is meant with Native here. A brief discussion on how the native graph is built with fallback to host-side execution could be elsewhere, maybe in the file comment?